lib/handle-create-event.js

```diff
@@ -3,21 +3,32 @@ module.exports = handleCreateEvent
 const readFileSync = require('fs').readFileSync
 const pathResolve = require('path').resolve
 
-const template = readFileSync(pathResolve(__dirname, '..', 'instructions.md'), 'utf8')
+const getDebug = require('debug')
 
 const getBranch = require('./get-branch')
 const getCommit = require('./get-commit')
 const createIssue = require('./create-issue')
 
+const template = readFileSync(pathResolve(__dirname, '..', 'instructions.md'), 'utf8')
+
 function handleCreateEvent (context) {
-  const {ref, refType, repository} = context.payload
+  const {ref, ref_type: refType, repository} = context.payload
```

Comment:
this was the problem, I must have made the mistake during my refactoring

Reply:
:1st_place_medal: